src/Microsoft.Health.Dicom.Web/roles.json

```diff
@@ -0,0 +1,25 @@
+﻿{
+    "$schema": "../Microsoft.Health.Dicom.Core/Features/Security/roles.schema.json",
+    "roles": [
+        {
+            "name": "globalReader",
+            "dataActions": [
+                "read"
+            ],
+            "notDataActions": [],
```

Comment:
Is this the expected schema? I am not sure if we will ever use NotDataActions

Reply:
Yes, this is the expected schema once defined for Azure RBAC - https://docs.microsoft.com/en-us/azure/role-based-access-control/role-definitions#notdataactions